charts/lab-partner/templates/secret.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  slackSigningSecret: {{ .Values.image.secrets.slackSigningSecret }}
+  slackBotUserOauthAccessToken: {{ .Values.image.secrets.slackBotUserOauthAccessToken }}
+  teamId: {{ .Values.image.secrets.teamId }}
```

Comment:
why are each of these secrets being pulled from `.Values.image`?  it would make more sense to me for them to come from `.Values.secrets` or something